zstacklib/zstacklib/utils/lv_metadata.py

```
@@ -0,0 +1,948 @@
+
+    def _extract_topology(d):
+        vols = {}
+        for rm in d.get('volumes', []):
+            vo_str = rm.get('vo', '{}')
+            try:
+                vo = json.loads(vo_str) if isinstance(vo_str, str) else vo_str
+                vols[vo.get('uuid', '')] = vo.get('installPath', '')
+            except (ValueError, TypeError):
+                pass
+        snaps = {}
+        for vol_uuid, snap_list in d.get('snapshots', {}).items():
```

Ruff (0.15.6)

[warning] Loop control variable `vol_uuid` not used within loop body

Rename unused `vol_uuid` to `_vol_uuid`

(B007)